setup.py

```diff
@@ -76,7 +76,7 @@ def read(fname):
     extras_require={
         'test': tests_require,
         's3': aws_deps,
-        'gcp': gcp_deps,
+        'gcs': gcp_deps,
```

Comment:
Please use GCS everywhere.

```suggestion
        'gcs': gcs_deps,
```